lms/static/js/student_account/views/AccessView.js

```diff
@@ -63,6 +63,11 @@
                     if ( ! window.isExternal( options.login_redirect_url ) ) {
                         this.nextUrl = options.login_redirect_url;
                     }
+
+                    if (options.origin) {
```

Comment:
What are you planning to do here, and why this approach? Write it down in comments please.